src/test/java/com/github/wovnio/wovnjava/ApiTest.java

```diff
@@ -1,143 +1,143 @@
-                                     "&token=token0" +
-                                     "&lang_code=ja" +
-                                     "&url_pattern=path" +
-                                     "&site_prefix_path=" +
-                                     "&custom_domain_langs=" +
-                                     "&product=wovnjava" +
-                                     "&version=" + Settings.VERSION +
-                                     "&debug_mode=false" +
-                                     "&body=" + html;
-        assertEquals(expectedRequestBody, apiRequestBody);
-
-        return result;
-    }
-
-    private byte[] gzip(byte[] input) throws IOException, ProtocolException {
-        ByteArrayOutputStream buffer = new ByteArrayOutputStream(input.length);
-        GZIPOutputStream gz = new GZIPOutputStream(buffer);
-        try {
-            gz.write(input);
-        } finally {
-            gz.close();
-        }
-        return buffer.toByteArray();
-    }
-
-    private HttpsURLConnection mockHttpsURLConnection(ByteArrayOutputStream requestStream, ByteArrayInputStream responseStream, int code, String encoding) throws IOException, ProtocolException {
-        HttpsURLConnection mock = EasyMock.createMock(HttpsURLConnection.class);
-        mock.setDoOutput(true);
-        mock.setRequestProperty(EasyMock.anyString(), EasyMock.anyString());
-        EasyMock.expectLastCall().atLeastOnce();
-        mock.setRequestMethod("POST");
-        EasyMock.expect(mock.getResponseCode()).andReturn(code);
-        EasyMock.expect(mock.getContentEncoding()).andReturn(encoding);
-        EasyMock.expect(mock.getOutputStream()).andReturn(requestStream);
-        EasyMock.expect(mock.getInputStream()).andReturn(responseStream);
-        EasyMock.replay(mock);
-        return mock;
-    }
-
-    private static String decompress(byte[] compressed) throws IOException {
-        StringBuilder sb = new StringBuilder();
-        ByteArrayInputStream bis = null;
-        GZIPInputStream gis = null;
-        BufferedReader br = null;
-        try {
-            bis = new ByteArrayInputStream(compressed);
-            gis = new GZIPInputStream(bis);
-            br = new BufferedReader(new InputStreamReader(gis, "UTF-8"));
-            String line;
-            while((line = br.readLine()) != null) {
-                sb.append(line);
-            }
-        } finally {
-            gis.close();
-            bis.close();
-            br.close();
-        }
-        return sb.toString();
-    }
-
-    private ResponseHeaders mockResponseHeaders() {
-        ResponseHeaders mock = EasyMock.createMock(ResponseHeaders.class);
-        mock.forwardFastlyHeaders(EasyMock.anyObject(HttpsURLConnection.class));
-        EasyMock.expectLastCall().times(1);
-        mock.setApiStatusCode("200");
-        EasyMock.expectLastCall().times(1);
-        EasyMock.replay(mock);
-        return mock;
-    }
-}
+// package com.github.wovnio.wovnjava;
```

Comment:
should this not be commented out?

Reply:
This branch was originally a debug build for a customer, but i'll definitely fix the tests when intending to merge it